at_client/lib/src/transformer/response_transformer/notification_response_transformer.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:async';
+
+import 'package:at_client/at_client.dart';
+import 'package:at_client/src/decryption_service/decryption.dart';
+import 'package:at_client/src/transformer/at_transformer.dart';
+import 'package:at_client/src/decryption_service/decryption_manager.dart';
+import 'package:meta/meta.dart';
+
+class NotificationResponseTransformer
```

Comment:
I'd like to see some unit tests on this new class please.

Reply:
I have added unit tests in notification_service_test.dart file. May i know if the comments mean to add few more test cases?